app/views/users/_form.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= form_for @user do |f| %>
+  <%= render "shared/error_messages" %>
+
+  <%= f.label :name %>
+  <%= f.text_field :name, class: "form-control" %>
+
+  <%= f.label :email %>
+  <%= f.email_field :email, class: "form-control" %>
+
+  <%= f.label :password %>
+  <%= f.password_field :password, class: "form-control" %>
+
+  <%= f.label :password_confirmation, t(".confirmation") %>
+  <%= f.password_field :password_confirmation, class: "form-control" %>
+
+  <%= f.submit yield(:button_text), class: "btn btn-primary" %>
```

Comment:
`yield` ở đây hoạt động như nào vậy bạn

Reply:
yeild ở đây có tác dụng chia block này ra. yeild nằm trong cặp <%= %> sẽ nhận giá trị của :button_text ( ở đây khi sử dụng ta cung cấp qua method provide) sau đó mới in ra.